docs/install.md

```diff
@@ -300,12 +298,12 @@ This image can be pushed to a container registry and shipped independently. All
 ## Kubernetes
 
 The setup for Kubernetes is very similar to that for Docker, and requires a fairly minimal deployment or pod definition to function. There
-are a few options to mix and match, including a deployment without a cache file, a stateful set with a persistant cache, and a standalone
+are a few options to mix and match, including a deployment without a cache file, a stateful set with a persistent cache, and a standalone
 unmanaged pod.
 
-
-=== "deployment"
-    ```yaml
+Deployment
+===
+  ```yaml
```

Comment:
While not correct Markdown, this syntax is is actually correct for `mkdocs`. It generates these abs that you see in the docs everywhere (example: https://ntfy.sh/docs/publish/).

So please revert this.

Reply:
Reverted to mkdocs syntax